mail_tracking/models/mail_tracking_email.py

```diff
@@ -91,14 +92,35 @@ class MailTrackingEmail(models.Model):
     tracking_event_ids = fields.One2many(
         string="Tracking events", comodel_name='mail.tracking.event',
         inverse_name='tracking_email_id', readonly=True)
+    # Token isn't generated here to have compatibility with older trackings.
+    # New trackings have token and older not
+    token = fields.Char(string="Request Token", readonly=True,
```

Comment:
This name is clearer:

```suggestion
    token = fields.Char(string="Security Token", readonly=True,
```

I'd add also `groups="base.group_system"`. We don't want regular users to forge an RPC request and read security tokens.